### DISCUSSION TITLE
Too many certificates requested. No more than 5 duplicate certs can be issued per week

### DISCUSSION THREAD

**Discussion by sqaisar** [vercel/community#2103] [Help]
I'm using custom domain and using custom domain for preview url's. I assign one specific alias to the PR (branch-name) it works well but if we push more that 5 commits to the same branch, it get deploys every time and I assign that same alias to the latest deployment every time it would fail after 5 times because of the duplicate cert request that vercel requests that's thrown by letsencrypt.

So in theory if we've already got an certificate for `feat-1.abc.example.com` vercel shouldn't be requesting the certificate for it again when the branch's deployment is updated and it tries to generate certificate every time even though it already got one for that specific name.

Is there a way to request wildcard certificate using Vercel?

**Comment by amyegan** [comment]
The error may be related to a recent incident: https://www.vercel-status.com/incidents/gmzr597mt95w. The incident was resolved, so I would expect things to be acting normally now.

If you still want to set up a wildcard certificate, you can [use a wildcard domain](https://vercel.com/docs/concepts/projects/domains/add-a-domain#using-wildcard-domain) to cover all subdomains under a single certificate. Note that wildcard domains must be configured with the [nameservers method](https://vercel.com/docs/concepts/projects/domains/add-a-domain#vercel-nameservers). For more info, please visit the following resources:
- [Subdomains, wildcard domains, and apex domains](https://vercel.com/docs/concepts/projects/domains#subdomains,-wildcard-domains,-and-apex-domains)
- [Why must we use the Domain Nameservers method for Wildcard Domains on Vercel?](https://vercel.com/guides/why-use-domain-nameservers-method-wildcard-domains)
- [How to Build a Multi-Tenant App with Custom Domains Using Next.js](https://vercel.com/guides/nextjs-multi-tenant-application)
